Friendly-plans/app/src/main/res/layout/fragment_plan_create.xml

```diff
@@ -49,16 +49,35 @@
         android:text="@={planData.planName}"
         tools:text="This is plan name." />
 
-      <!--Next btn-->
-      <Button
-        android:id="@+id/id_btn_plan_next"
+      <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/id_layout_plan_add_elements"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_column="4"
-        android:layout_row="5"
-        android:onClick="@{() -> planDataClick.savePlanData(planData)}"
-        android:text="@string/next" />
+        android:layout_column="0"
+        android:layout_row="3"
+        android:orientation="vertical">
+
+        <Button
+          android:id="@+id/id_btn_plan_create_tasks"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:onClick="@{() -> planDataClick.savePlanData(planData, 1)}"
+          android:text="@string/create_plan_add_tasks_type_1" />
+        <Button
+          android:id="@+id/id_btn_plan_create_prizes"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:onClick="@{() -> planDataClick.savePlanData(planData, 2)}"
+          android:text="@string/create_plan_add_tasks_type_2" />
+        <Button
+          android:id="@+id/id_btn_plan_create_interactions"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:onClick="@{() -> planDataClick.savePlanData(planData, 3)}"
+          android:text="@string/create_plan_add_tasks_type_3" />
```

Comment:
Nice. It good you used binding! 👍 Could you please rename the string names from numbers to says what it is? create_plan_add_tasks_type_interaction. I believe it will be easier to maintain changes in the future. It would be also nice to use enum in planDataClick.savePlanData(planData, TaskType.INTERACTION) here, but I am not sure it is possible.